src/commands/orders.py

```diff
@@ -66,137 +69,151 @@ def to_order_dto(order):
       format_integer(order['order_id']) + '\n' + 
 
-      click.style('  From batch', fg=labelColor) + ': ' + 
-      format_batch_id_with_date(order['fromBatchId']) + '\n' +
+      click.style('  From batch', fg=label_color) + ': ' + 
+      format_batch_id_with_date(order['from_batch_id']) + '\n' +
 
-      click.style('  To batch', fg=labelColor) + ': ' +
-      format_batch_id_with_date(order['untilBatchId']) + 
+      click.style('  To batch', fg=label_color) + ': ' +
+      format_batch_id_with_date(order['until_batch_id']) + 
       '\n\n' + 
 
-      click.style('  Sell Token', fg=labelColor) + ': ' + 
-      format_token_long(sellToken) + '\n' + 
+      click.style('  Sell Token', fg=label_color) + ': ' + 
+      format_token_long(sell_token) + '\n' + 
 
-      click.style('  Buy Token', fg=labelColor) + ': ' + 
-      format_token_long(buyToken) + '\n' +
+      click.style('  Buy Token', fg=label_color) + ': ' + 
+      format_token_long(buy_token) + '\n' +
 
-      click.style('  Sold volume', fg=labelColor) + ': ' + 
-      format_amount_in_weis(soldVolume, sellTokenDecimals) +
+      click.style('  Sold volume', fg=label_color) + ': ' + 
+      format_amount_in_weis(sold_volume, sellTokenDecimals) +
       ' of ' +
-      format_amount_in_weis(maxSellAmount, sellTokenDecimals) + ' ' + sellTokenLabel + 
+      format_amount_in_weis(max_sell_amount, sellTokenDecimals) + ' ' + sellTokenLabel + 
       percentageText +
       '\n' + 
 
       (
-        click.style('  Bought volume', fg=labelColor) + ': ' + 
-        format_amount_in_weis(boughtVolume, buyTokenDecimals) + ' ' + buyTokenLabel +      
+        click.style('  Bought volume', fg=label_color) + ': ' + 
+        format_amount_in_weis(bought_volume, buyTokenDecimals) + ' ' + buyTokenLabel +      
         '\n'
-        if soldVolume else ''
-      ) + # TODO: Add percentage https://github.com/gnosis/dex-cli/issues/31
+        if sold_volume else ''
+      ) +
       '\n' +
 
-      click.style(f'  Limit Price {sellTokenLabel}/{buyTokenLabel}', fg=labelColor) + ': ' + 
-      format_price(calculate_price(
-        numerator=priceNumerator,
-        denominator=priceDenominator,
-        decimals_numerator=buyTokenDecimals,
-        decimals_denominator=sellTokenDecimals
-      ), currency=buyTokenLabel) + '\n' +      
-
-      click.style(f'  Limit Price {buyTokenLabel}/{sellTokenLabel}', fg=labelColor) + ': ' + 
-      format_price(calculate_price(
-        numerator=priceDenominator,
-        denominator=priceNumerator,
-        decimals_numerator=sellTokenDecimals,
-        decimals_denominator=buyTokenDecimals
-      ), currency=sellTokenLabel) + '\n' +
-
-      tradePriceText +
+      # Prices
+      f'{order_price_text_1}\n{order_price_text_2}\n' +
+      trade_price_text +
       '\n' +
 
-      click.style('  Transaction', fg=labelColor) + ': ' + 
+      click.style('  Transaction', fg=label_color) + ': ' + 
       to_etherscan_link(order['txHash']) + '\n' + 
```

Comment:
Still just a nit: I'd prefer to have the same style throughout. 
Here in the same line there's a mix of f-string and string concatenation. Pick one :)
Or don't.

Reply:
You are right! The reason I did it is because I thought it was more readable-mantainable for `print_trades_pretty` to just concatenate all the parts of the message (so use concatenation), BUT some of the parts more concise in f-strings. 
I'm happy to review it overall if you feel is too bad :)

Reply:
Well, it does bother me, but I can live with that. (until I get to this code :smiling_imp: )